include/secp256k1.h

```diff
@@ -97,7 +97,10 @@ typedef struct {
     uint64_t magic;
     /* Public nonce before applying the sign-to-contract commitment */
     secp256k1_pubkey original_pubnonce;
-    /* Integer indicating if signing algorithm negated the nonce */
+    /* Byte indicating if signing algorithm negated the nonce. Alternatively when
+     * verifying we could compute the EC commitment of original_pubnonce and the
+     * data and negate if this would not be a valid nonce. But this would prevent
+     * batch verification of sign-to-contract commitments. */
     unsigned char nonce_is_negated;
```

Comment:
nit, feel free to ignore: Boolean values are typically represented by `int` and not `unsigned char`.

Reply:
Yes but ints are the worst. Since `nonce_is_negated` is only ever 0 or 1 I changed this to int.